Libraries/Components/Picker/Picker.js

```diff
@@ -115,6 +120,10 @@ type PickerProps = $ReadOnly<{|
    * The string used for the accessibility label. Will be read once focused on the picker but not on change.
    */
   accessibilityLabel?: ?string,
+
+  accessible?: ?boolean,
```

Comment:
We should add a comment for each prop as well as the other props do

Reply:
Okay, I will improve this!